internal/guest/storage/scsi/scsi.go

```diff
@@ -168,16 +168,13 @@ func Mount(
 			// The `source` found by controllerLunToName can take some time
 			// before its actually available under `/dev/sd*`. Retry while we
 			// wait for `source` to show up.
-			if errors.Is(err, unix.ENOENT) {
-				select {
-				case <-ctx.Done():
-					return ctx.Err()
-				default:
-					time.Sleep(10 * time.Millisecond)
-					continue
-				}
+			select {
+			case <-ctx.Done():
+				return ctx.Err()
```

Comment:
actually, one comment here: with this change we're throwing away the error code from the unixMount call. Can we try to at least log that still?

Reply:
Good point. Updated!